code/game/objects/items/devices/laserpointer.dm

```diff
@@ -131,7 +131,7 @@
 
 	//catpeople
 	for(var/mob/living/carbon/human/H in view(1,targloc))
-		if(!iscatperson(H) || H.incapacitated() || H.eye_blind )

```

Comment:
this entire block of code needs to be removed, by doing this you're making it so all people are affected by laser pointers

Reply:
we're all part cat on the inside

Reply:
Well that was a hilarious oversight